setup.py

```diff
@@ -30,6 +30,7 @@
         'Framework :: Django :: 1.9',
         'Framework :: Django :: 1.10',
         'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.0',
```

Comment:
Django 2.2 LTS was just released today. Worth adding support?

Reply:
Hmm, I think so. I think I'll merge this then test 2.1 and 2.2. 
If everything works I'll just add the versions this list. 
If it doesn't I'll make an issue for whatever changes are needed.